### DISCUSSION TITLE
Make bug report template link to Q&A?

### DISCUSSION THREAD

**Discussion by spenserblack** [lowlighter/metrics#561] [Ideas]
I noticed that some bug reports probably belong as Q&A instead. Perhaps bug reports should also be Q&A discussions? They can be converted to issues later if they turn out to be a valid bug and not an issue with the user's config.

**Comment by lowlighter** [comment]
Yeah I'll consider changing that in the near future 👍 
Thanks for the suggestion!

**Reply by lowlighter** [reply]
This has been changed, Issues cannot be created directly now and requires to be first opened through discussions
